### DISCUSSION TITLE
Can you have multiple resources for one citation? For example, 3 images and 2 pdf files for one citation?

### DISCUSSION THREAD

**Discussion by AbeEtan** [emacs-citar/citar#691] [Q&A] [answered]

**Comment by bdarcus** [comment]
Yes. Should be documented in the README.

**Comment by AbeEtan** [comment]
What if I wanted to link two or more images, how would I be able to do that? For instance, I have a citation in my lib called 

```
@book{myers_2021,
  place =	 {Maine, USA},
  edition =	 {4th},
  title =	 {Anatomy trains: Myofascial Meridians for manual
                  therapists and Movement Professionals},
  publisher =	 {Elsevier Health Sciences},
  author =	 {Myers, Thomas W.},
  year =	 2021
}
```

I named the pdf `myers_2021.pdf`. Now I want to add a few images from the book, such as `myers_2021.jpeg`, what would I have to name the remaining images?

**Comment by bdarcus** [comment, marked as answer]
You can add this to your config, where an example filename would be `test-1.jpeg`?

```elisp
(setq citar-file-additional-files-separator "-") ; use whatever separator you want here
```

**Comment by AbeEtan** [comment]
![Screenshot from 2022-09-12 15-55-55](https://user-images.githubusercontent.com/85316227/189668366-8fe46a29-5152-4030-a53f-2de965e642f2.png)

As you can see, I can see the other links without adding any code to my config file, but what if I wanted to add more than one image? I can't have duplicated items in the same folder if that makes sense, what is the right way to do it?

**Reply by bdarcus** [reply]
I edited [this](https://github.com/emacs-citar/citar/discussions/691?converting=1#discussioncomment-3625976) after the fact; make sure you reread the edited version carefully ;-)

**Reply by AbeEtan** [reply]
![image](https://user-images.githubusercontent.com/85316227/189670415-e304d4b2-b7f7-4640-baa2-c71124d023de.png)

Is this what you mean?

**Reply by bdarcus** [reply]
Yes, exactly.

**Reply by AbeEtan** [reply]
I still only see the two links. I don't see the other remaining links. I just tried it with another reference and It worked!

Maybe there is something with the myers reference.

**Reply by bdarcus** [reply]
And you set the variable, per my example?

**Reply by AbeEtan** [reply]
![image](https://user-images.githubusercontent.com/85316227/189672139-4e8c9508-067d-42e1-aee7-1905297d5a92.png)

**Reply by AbeEtan** [reply]
Does the file size have to do with it not working?

**Reply by bdarcus** [reply]
Great; I'm updating the README in a minute.

**Reply by AbeEtan** [reply]
Please ignore that, such a silly mistake on my end, the files were not in the same folder. It's working like a charm. Thank you so much!

**Reply by bdarcus** [reply]
> Does the file size have to do with it not working?

WDYM "not working"? You mean by default (because your screenshots shows it's working; right?)?

If yes, it's just theoretically slower to scan for the additional files, I think.

Also, users have different approaches to file naming, so better to leave it off by default.

**Reply by AbeEtan** [reply]
![image](https://user-images.githubusercontent.com/85316227/189673306-4cb9a9b4-1922-4ee6-a012-fba86db0798b.png)
See. It works beautifully. Thank you from the bottom of my heart!

**Reply by AbeEtan** [reply]
Yes, sorry it was a mistake on my end, the files were not in the same folder. It works perfectly now! I attached the successful image above.

**Reply by deen1** [reply]
Sorry, how was it that you were able to add another `.pdf` file for an entry? I have set `citar-file-additional-files-separator`, but when I try to do `M-x citar-add-file-to-library` for an entry that already has a file, I get given the "File ... exists, overwrite? (y or n)" prompt in the minibuffer.
